### PR TITLE
feat(manifests): update plugin manifest to run in dedicated environment and custom NS

### PR DESCRIPTION
**Pull request by mtulio**
Plugin implementation: https://github.com/redhat-openshift-ecosystem/provider-certification-plugins/pull/3

### CODE REVIEW

**Comment by bostrt** [comment]
/lgtm cancel
Removing since we found some issues with tolerations.

**Comment by mtulio** [comment]
@bostrt do you want to keep all changes* on this PR?

*Currently changes/issues found:
1. Tolerations need to be added to the label `scheduler.alpha.kubernetes.io/defaultTolerations` of NS
2. `podSpec.resources.requests` are impacting in `--dedicated` environments running in minimal compute resources (8GiB RAM)

**Comment by bostrt** [comment]
> @bostrt do you want to keep all changes* on this PR?

Yep, I'm okay with keeping it all in this PR!

File: pkg/run/run.go [hunk 1]
```diff
@@ -262,8 +262,8 @@ func (r *RunOptions) Run(kclient kubernetes.Interface, sclient sonobuoyclient.In
 			ObjectMeta: metav1.ObjectMeta{
 				Name: pkg.CertificationNamespace,
 				Annotations: map[string]string{
-					v1.TolerationsAnnotationKey:  string(tolerations),
-					"openshift.io/node-selector": "node-role.kubernetes.io/tests=",
+					"scheduler.alpha.kubernetes.io/defaultTolerations": string(tolerations),
```
Comment (mtulio) [inline pkg/run/run.go:265]: @bostrt I may be wrong here but didn't find the "defaultTolerations" object.

Comment (bostrt) [reply pkg/run/run.go:265]: It took me a while but I did end up finding it yesterday here: https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/plugin/pkg/admission/podtolerationrestriction/admission.go#L56-L60

I'm okay with hardcoding this string into our tool so we don't have to pull in such a large dependency for a single string.

**Pull request merged**